### PR TITLE
feat: added Watch later and nav

### PR DESCRIPTION
**Pull request by Chakravarthi0**
added watch later
made navbar responsive

### CODE REVIEW

File: src/layouts/aside-navbar/AsideNavBar.jsx [hunk 1]
```diff
@@ -1,42 +1,90 @@
...
         </li>
         <li>
-          <Link className="link aside-nav-link" to={"/history"}>
+          <NavLink
+            className={({ isActive }) =>
+              isActive
+                ? "link aside-nav-link active-aside-link"
+                : "link aside-nav-link gray"
+            }
+            to={"/history"}
```
Comment (msdeshmukh009) [inline src/layouts/aside-navbar/AsideNavBar.jsx:83]: You don't need curly brackets here.

File: src/layouts/navbar/navbar.css [hunk 2]
```diff
@@ -1,13 +1,52 @@
...
 
+.side-navbar {
+  position: fixed;
+  z-index: 2;
+  inset: 0;
+  padding-top: 1rem;
+  height: 100vh;
+  width: 60%;
+  background-color: var(--primary-color);
+  background-color: var(--white-pure);
```
Comment (msdeshmukh009) [inline src/layouts/navbar/navbar.css]: Repeating.

File: src/pages/watch-later/WatchLater.jsx [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
-import { VideoCard } from "../../components/index";
+import { VideoCard, LoadingError, Loader } from "../../components/index";
 import { videos } from "../../backend/db/videos";
+import { useWatchLater } from "../../context/index";
 import "./watch-later.css";
 
 function WatchLater() {
+  const {
+    watchLaterState: { watchLater, loading, error },
+    addToWatchLater,
```
Comment (msdeshmukh009) [inline src/pages/watch-later/WatchLater.jsx]: You don't need this I guess

**Review by msdeshmukh009 — APPROVED** [review]
The code looks clean and UI is working as expected. Good work!!

File: src/layouts/aside-navbar/AsideNavBar.jsx [hunk 4]
```diff
@@ -1,42 +1,90 @@
...
         <li>
-          <Link className="link aside-nav-link" to={"/playlists"}>
+          <NavLink
+            className={({ isActive }) =>
+              isActive
+                ? "link aside-nav-link active-aside-link"
+                : "link aside-nav-link gray"
+            }
+            to={"/playlists"}
+          >
```
Comment (poornima-alugubelly) [inline src/layouts/aside-navbar/AsideNavBar.jsx:44]: as mentioned by Mahesh u can remove curly braces in `to`

**Review by poornima-alugubelly — APPROVED** [review]
clean code good work 👍

File: src/layouts/navbar/navbar.css (hunk 2 above)
Comment (Chakravarthi0) [reply src/layouts/navbar/navbar.css]: removed.

**Pull request merged**